source/virtualBuffers/MSHTML.py

```diff
@@ -122,6 +122,7 @@ def _normalizeControlField(self,attrs):
 			# MSHTML puts the unavailable state on all graphics when the showing of graphics is disabled.
 			# This is rather annoying and irrelevant to our users, so discard it.
 			states.discard(controlTypes.STATE_UNAVAILABLE)
+		ariaRoles.append(aria.htmlNodeNameToAriaLandmarkRoles.get(nodeName.lower()))

```

Comment:
nit: `aria.htmlNodeNameToAriaLandmarkRoles.get` might return `None` if there was no matching key. Having `None` in the `ariaRoels` list doesn't break anything right now, but it is odd and it might be confusing in future. Perhaps something like this:

```
lRole = aria.htmlNodeNameToAriaLandmarkRoles.get(nodeName.lower()))
if lRole:
	ariaRoles.append(lRole)
```